src/libraries/System.Diagnostics.Process/ref/System.Diagnostics.Process.cs

```diff
@@ -43,8 +43,8 @@ public partial class Process : System.ComponentModel.Component, System.IDisposab
         public System.Diagnostics.ProcessModule? MainModule { get { throw null; } }
         public System.IntPtr MainWindowHandle { get { throw null; } }
         public string MainWindowTitle { get { throw null; } }
-        public System.IntPtr MaxWorkingSet { get { throw null; } [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")] [System.Runtime.Versioning.SupportedOSPlatformAttribute("macos")] [System.Runtime.Versioning.SupportedOSPlatformAttribute("freebsd")] set { } }
-        public System.IntPtr MinWorkingSet { get { throw null; } [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")] [System.Runtime.Versioning.SupportedOSPlatformAttribute("macos")] [System.Runtime.Versioning.SupportedOSPlatformAttribute("freebsd")] set { } }
+        public System.IntPtr MaxWorkingSet { [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("ios"), System.Runtime.Versioning.UnsupportedOSPlatformAttribute("tvos")] get { throw null; } [System.Runtime.Versioning.SupportedOSPlatformAttribute("freebsd"), System.Runtime.Versioning.SupportedOSPlatformAttribute("macos"), System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")] set { } }
```

Comment:
Hmm, this should not be needed.

/cc @buyaa-n

Reply:
> Hmm, this should not be needed.

Right, the existing supported attributes expressing the API is only supported on `"freebsd", "macos" and "windows"` and unsupported all other, adding `UnsupportedOSPlatform("ios"), UnsupportedOSPlatform("tvos")` is redundant

Reply:
So if those supported attributes annotate the setters only - will the getters be marked automatically? My change is to annotate the latter ones.

Reply:
Oh sorry, i didn't notice that it was separate attributes for getter and setter, you are right